deepchecks/tabular/checks/model_evaluation/train_test_prediction_drift.py

```diff
@@ -186,15 +255,26 @@ def add_condition_drift_score_less_than(self, max_allowed_categorical_score: flo
                 max_allowed_numeric_score = max_allowed_earth_movers_score
 
         def condition(result: Dict) -> ConditionResult:
-            drift_score = result['Drift score']
+            drift_score_dict = result['Drift score']
+            # Move to dict for easier looping
+            if not isinstance(drift_score_dict, dict):
+                drift_score_dict = {0: drift_score_dict}
             method = result['Method']
-            has_failed = (drift_score >= max_allowed_categorical_score and method in SUPPORTED_CATEGORICAL_METHODS) or \
-                         (drift_score >= max_allowed_numeric_score and method in SUPPORTED_NUMERIC_METHODS)
+            has_failed = {}
+            drift_score = 0
+            for class_name, drift_score in drift_score_dict.items():
+                has_failed[class_name] = \
+                    (drift_score >= max_allowed_categorical_score and method in SUPPORTED_CATEGORICAL_METHODS) or \
+                    (drift_score >= max_allowed_numeric_score and method in SUPPORTED_NUMERIC_METHODS)
 
-            details = f'Found model prediction {method} drift score of {format_number(drift_score)}'
-            category = ConditionCategory.FAIL if has_failed else ConditionCategory.PASS
+            if len(has_failed) == 1:
+                details = f'Found model prediction {method} drift score of {format_number(drift_score)}'
+            else:
+                details = f'Found {sum(has_failed.values())} classes with model predicted probability {method} drift' \
+                          f' score above threshold: {format_number(max_allowed_numeric_score)}.'
+            category = ConditionCategory.FAIL if any(has_failed.values()) else ConditionCategory.PASS
             return ConditionResult(category, details)
 
-        return self.add_condition(f'categorical drift score < {max_allowed_categorical_score} and '
-                                  f'numerical drift score < {max_allowed_numeric_score}',
+        return self.add_condition(f'categorical drift score < {format_number(max_allowed_categorical_score)} and '
```

Comment:
this value is given by the user, is the format number really needed? i think it may cause confusion

Reply:
Good point, will change